src/07-projects/projects.apib

```diff
@@ -226,3 +226,18 @@ Update a participant's role for a project.
                 + member
 
 + Response 204
+
+
+### projects.updateInternalHourlyCost [POST /projects.updateInternalHourlyCost]
+
+Updates the internal hourly cost for a participant for one project, or all projects.
+This does not update the global internal rate of the employee. See `users.updateInternalHourlyCost`.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `082e6289-30c5-45ad-bcd0-190b02d21e81` (string, optional) - If there is no specific project id, update for all projects
```

Comment:
This should be required, we update the internal rate for a participant for one project. That would be consistent with the other endpoints.

Reply:
But how would you update the internal rate for all projects (for a deactivated/deleted user)?